valusets/ValueSet-England-ImmunizationExplanationReason.xml

```diff
@@ -0,0 +1,43 @@
+<ValueSet xmlns="http://hl7.org/fhir">
+    <id value="England-ImmunizationExplanationReason" />
+    <url value="https://fhir.nhs.uk/ValueSet/England-ImmunizationExplanationReason" />
+    <version value="1.0.0" />
+    <name value="EnglandImmunizationExplanationReason" />
+    <title value="England Immunization Explanation Reason" />
+    <status value="draft" />
+    <date value="2023-06-01" />
+    <publisher value="NHS England" />
+    <contact>
+        <name value="NHS England" />
+        <telecom>
+            <system value="email" />
+            <value value="interoperabilityteam@nhs.net" />
+            <use value="work" />
+            <rank value="1" />
+        </telecom>
+    </contact>
+    <description value="A set of codes that define a clinical indication or reason for administering a vaccine. Selected from the &#39;Healthcare matters&#39; simple reference set of the SNOMED CT UK coding system." />
+    <copyright value="Copyright &#169; 2021+ NHS Digital Licensed under the Apache License, Version 2.0 (the &quot;License&quot;); you may not use this file except in compliance with the License. You may obtain a copy of the License at  http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an &quot;AS IS&quot; BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License. HL7&#174; FHIR&#174; standard Copyright &#169; 2011+ HL7 The HL7&#174; FHIR&#174; standard is used under the FHIR license. You may obtain a copy of the FHIR license at  https://www.hl7.org/fhir/license.html.&lt;br$gt;This value set includes content from SNOMED CT, which is copyright &#169; 2002+ International Health Terminology Standards Development Organisation (IHTSDO), and distributed by agreement between IHTSDO and HL7. Implementer use of SNOMED CT is not covered by this agreement." />
+    <compose>
+        <include>
+            <system value="http://snomed.info/sct" />
+            <filter>
+                <property value="constraint" />
+                <op value="=" />
+                <value value="memberOf 1127601000000107" />
+            </filter>
+        </include>
+        <include>
+            <system value="http://snomed.info/sct" />
+            <concept>
+                <code value="429060002" />
+            </concept>
+            <concept>
+                <code value="281657000" />
```

Comment:
This is a retired concept - do we want to include it?

Reply:
Replaced with 171072005